src/digitalIdentity-backend/src/main/java/didentity/amos/digitalIdentity/services/LissiApiService.java

```diff
@@ -1,7 +1,7 @@
 package didentity.amos.digitalIdentity.services;
 
-import java.io.File;
```

Comment:
The backend doesn't compile because of this deletion...

Reply:
l added it again